packages/Python/lldbsuite/test/lang/swift/implementation_only_imports/library_indirect/TestLibraryIndirect.py

```diff
@@ -12,7 +12,14 @@
 """
 Test `@_implementationOnly import` behind some indirection in a library used by the main executable
 """
-import commands
+
+#SWIFT_TENSORFLOW: Since tensorflow uses Python 3 and commands is deprected
+# there, import subprocess instead.
+try
```

Comment:
this should be `try:`?

Reply:
For some reason it worked without it but adding it anyway for consistency.